### PR TITLE
feat: Add to expenses to report section (Use Exact Amount)

### PR DESCRIPTION
**Pull request by SahilK-027**
## Clickup
[clickup link](https://app.clickup.com/t/86cx5qh88)

## UI Preview
Screenshot 2024-12-10 at 4 58 54 PM
Screenshot 2024-12-10 at 4 58 43 PM

## Summary by CodeRabbit

- **New Features**
	- Enhanced visual representation in the add transaction dialog with new icons and layout adjustments.
	- Introduced a new currency formatting option for report amounts.

- **Bug Fixes**
	- Improved styling consistency across various elements in the report list and my expenses sections.

- **Documentation**
	- Updated styles for better visual hierarchy and spacing in footer and button sections.

- **Tests**
	- Added new pipe to the test suite for better currency formatting validation.

### CODE REVIEW

File: src/global.scss [hunk 1]
```diff
@@ -87,7 +87,7 @@ ion-popover.error-popover {
 }
 
 .mat-bottom-sheet-1 {
-  padding: 24px 16px !important;
+  padding: 16px 12px 12px 12px !important;
```
Comment (Dimple16) [inline src/global.scss:90]: won't this impact all the screens?

File: src/app/fyle/my-expenses/add-txn-to-report-dialog/add-txn-to-report-dialog.component.spec.ts [hunk 2]
```diff
@@ -15,8 +15,9 @@ import { ReportState } from 'src/app/shared/pipes/report-state.pipe';
 import { SnakeCaseToSpaceCase } from 'src/app/shared/pipes/snake-case-to-space-case.pipe';
 import { AddTxnToReportDialogComponent } from './add-txn-to-report-dialog.component';
 import { expectedReportsSinglePage } from 'src/app/core/mock-data/platform-report.data';
+import { ExactCurrencyPipe } from 'src/app/shared/pipes/exact-currency.pipe';
 
-describe('AddTxnToReportDialogComponent', () => {
+fdescribe('AddTxnToReportDialogComponent', () => {
```
Comment (Dimple16) [inline src/app/fyle/my-expenses/add-txn-to-report-dialog/add-txn-to-report-dialog.component.spec.ts:20]: remove `f`

File: src/app/fyle/my-expenses/add-txn-to-report-dialog/add-txn-to-report-dialog.component.scss [hunk 3]
```diff
@@ -3,25 +3,36 @@ $reports_sent_back_color: #da1e28;
...
 
     &--row-icon-container {
       display: flex;
       align-items: center;
       justify-content: center;
+      font-size: 20px;
+    }
+
+    &--handle-bar-icon {
+      margin-top: 0px;
```
Comment (Dimple16) [inline src/app/fyle/my-expenses/add-txn-to-report-dialog/add-txn-to-report-dialog.component.scss:20]: combine these both

**Review by Dimple16 — CHANGES_REQUESTED** [review]
added comments

File: src/global.scss (hunk 1 above)
Comment (SahilK-027) [reply src/global.scss:90]: Nope it won't affect other screens, we are using this pannel only for `AddTxnToReportDialogComponent`

Screenshot 2024-12-10 at 5 30 17 PM

**Review by Dimple16 — APPROVED** [review]

**Pull request merged**